src/rdkafka_op.h

```diff
@@ -394,8 +396,13 @@ struct rd_kafka_op_s {
                                             *   that needs to speak to a
                                             *   specific broker rather than
                                             *   the controller.
-                                            *   Defaults to -1:
-                                            *   look up and use controller. */
+                                            *   Special values:
+                                            *   -1:
+                                            *   look up and use controller.
+                                            *   -2:
+                                            *   look up and use coordinator.
+                                            *   */
+                        char *coordname; /**< Which coordinator to look up.  */
```

Comment:
Looking at rdkafka_coord.c we should probably have both coordtype and coordkey here.